starter-code/database.js

```diff
@@ -54,11 +54,10 @@ class Database {
       if (error){
         callback(error);
       } else {
-        //  LAB 2
-        // Implement the query to insert a user
-        // remeber once it's finish to comment callback('Error listing users');
+
+        database.collection('users').find(callback);
         
-        callback('Error listing users');
+        //callback('Error listing users');
```

Comment:
you're missing deleteUser which should be
        **database.collection('users').deleteOne(users.firstName, callback);**